### DISCUSSION TITLE
免密码登录

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2974] [高级账户]
# 重要提示

> 嗨，简悦全新的 [教程中心](https://kb.simpread.pro/) 已上线，**下面的内容已经已同步更新**，建议 [访问这里](https://kb.simpread.pro/#/page/免密码登录) 查看。

***

# 描述

> 因为不收集用户信息，所以简悦最大的特色之一：**无账户系统，只有凭证系统**，虽然在登录时提供多种登录方式：

1. UID / License

2. 坚果云账户 / 密码

3. 用户名 / 密码

3. 配置文件

但还是有用户希望可以再简化登录方案。因此，简悦再增加一种新的登录方式：

> **无密码验证 Email 登录方案**，目前包含的平台：

- [开放平台](https://github.com/Kenshin/simpread/discussions/2757) 以及相关功能，如：[个人中心](https://github.com/Kenshin/simpread/discussions/2959) · [阅读列表](https://github.com/Kenshin/simpread/discussions/2954) · [收藏助手](https://github.com/Kenshin/simpread/discussions/2921) 等。

- [UID 找回平台](https://simpread.pro/reset)

> 在新的扩展端，也会集成此功能。（**简悦 2.2.0.520 及以下版本暂未提供**）

# 注册你的 Email

> 只要在 **选项页 → 账户 → 用户信息** 提交你的 Email 地址即可。

![image](https://user-images.githubusercontent.com/81074/139421907-44a58098-3b87-4872-ab2c-03ad9d271890.png)

#### 注意事项

1. 不需验证，填写时请勿填写错误
2. 请在高级账户环境下填写

# 如何使用

> 例如登录到 https://simpread.pro/developer

![image](https://user-images.githubusercontent.com/81074/139519417-f3140d27-b6b0-485f-89f9-1a1d25538b17.png)

![image](https://user-images.githubusercontent.com/81074/139519433-354ea85a-7be2-4998-a06b-c4b70f1d1acd.png)

> 稍等片刻后就会收到验证邮件（如果没有请查看垃圾箱并将 `notifications@simpread.pro` 加入到信任列表（或白名单）

![image](https://user-images.githubusercontent.com/81074/139519457-0980442e-cd4c-448c-85ad-f6a3fae7e8cd.png)

> 将得到的登录验证码输入即可完成验证。

![image](https://user-images.githubusercontent.com/81074/139519494-865854fd-b3a3-4ded-b78c-75b35eb8cc3f.png)

> 如果输入无误的话，说明身份验证成功。

![image](https://user-images.githubusercontent.com/81074/139519567-4f35fa18-081d-4078-baf5-c2e6c06a67f2.png)

# 注意事项

1. 此方式为免密码登录，所以 Email 只是一个身份凭证，因此可以随意更换 Email 地址。
2. 如果是通过此方式登录 [开放平台](https://github.com/Kenshin/simpread/discussions/2079) 的话，仍需使用遵循开放平台的 [开通条件](https://github.com/Kenshin/simpread/discussions/2757)

**Comment by Kenshin** [comment]
## 422 错误

> 同一个 Email 地址对应了对一个 高级账户 UID

## 解决方案

> 为每一个 UID 配置一个不同的邮箱地址。

## 如何修改 Email

>  在 **选项页 → 账户 → 用户信息** 提交你的 Email 地址即可。（确保当前为你需要使用的 UID，细节可以看最上面的教程）

**Comment by Kenshin** [comment]
## 403 错误

> 提交的 Email 对应的 UID 不是高级账户。

## 解决方案

> 在高级账户环境下，重新提交一次 Email 地址，即可。（确保提交 Email 时为高级账户，细节可以看最上面的教程）

**Reply by davidxu5945** [reply]
选项页 → 账户 → 用户信息，里本来就有邮箱，我又重新保存了一遍。再次操作邮箱登陆，提示以下内容
![image](https://user-images.githubusercontent.com/3996007/142866484-0d385679-7391-4bee-97f0-70f7cde9a64c.png)

希望可以得到帮助

**Reply by Kenshin** [reply]
请问两次的邮箱是一个吗？

这个提示说明你提交的邮箱对应的 UID 不是高级账户。

最简单的方式：用高级账户对应的配置文件提交你的邮箱地址。

如果还是不行，你可以把 UID 与你认为正确的邮箱发到 help#simpread.pro 并附上 Issues 2947

**Reply by davidxu5945** [reply]
解决了，我现在填写的邮箱不是购买时收注册码的邮箱，换成购买时收注册码的邮箱，就可以了

**Reply by Kenshin** [reply]
解决就好，这说明，当时你提交的是购买时的邮箱。

别忘记在扩展端修改邮箱为正确的。

**Comment by BopSun** [comment]
我是从荔枝商城购买的，扩展端修改了没用，详细情况写邮件给你了。

**Reply by Kenshin** [reply]
嗨，我收到你的邮件了，不过在这回复下。

1. 如果是首次使用还是需要用 UID / License 登录的。
2. 免密码登录需要设置，并不是直接就能使用的，教程就在上方，或者 [看这里](https://kb.simpread.pro/#/page/%E5%85%8D%E5%AF%86%E7%A0%81%E7%99%BB%E5%BD%95)。

**Reply by BopSun** [reply]
我等了两天都没收到任何邮件，到这里才看到回复。

简单说一下我遇到的情况，除了我注册的这个邮箱以外，随便我修改任何邮箱都好使。gmail可以发送邮件，但是我收不到。不得已我只能改成QQ邮箱了。

**Reply by Kenshin** [reply]
> 我等了两天都没收到任何邮件，到这里才看到回复。
> 
> 简单说一下我遇到的情况，除了我注册的这个邮箱以外，随便我修改任何邮箱都好使。gmail可以发送邮件，但是我收不到。不得已我只能改成QQ邮箱了。

我不是说了吗？我在这里回复你了。https://github.com/Kenshin/simpread/discussions/2974#discussioncomment-2474494

**Comment by Darkluna999** [comment]
少数派买的。。之前收license的邮箱忘记了。。。这个邮箱能修改更新吗

**Reply by Kenshin** [reply]
可以的。

**Reply by davidxu5945** [reply]
我也是少数派买的，少数派的订单详情里 是有邮箱记录的

**Comment by sylph125** [comment]
用爱发电买的没收到UID 点邮件也没反应

**Reply by Kenshin** [reply]
使用注册时的手机号或 Email 登录你的爱发电查看私信。
